x/ununifidist/module.go

```diff
@@ -163,7 +163,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (AppModule) ConsensusVersion() uint64 { return 2 }
+func (AppModule) ConsensusVersion() uint64 { return 1 }
```

Comment:
Maybe this is unnecessary.

Reply:
@mkXultra 
Sorry to be presumptuous, but it seems that if the ConsensusVersion of each module is 2 or higher, you have to give them RegisterMigration at RegisterServices.
If the ConsensusVersion is 1, it seems that the version check is skipped when upgrading, but if the ConsensusVersion is 2 or more, it seems that the existence check is performed for each version.
Currently, that check results in an error during upgrade.

Should I even use this ticket to handle RegisterMigration, etc.? Please let me consult with you.

## cosmos-sdk@v0.45.6   
- crisis : 
https://github.com/cosmos/cosmos-sdk/blob/f2d94445c0f5f52cf5ed999b81048b575de94964/x/crisis/module.go#L133
```
func (am AppModule) RegisterServices(cfg module.Configurator) {
	types.RegisterMsgServer(cfg.MsgServer(), am.keeper)
}
...
func (AppModule) ConsensusVersion() uint64 { return 1 }
```

- bank : 
https://github.com/cosmos/cosmos-sdk/blob/cc5fe49ce7e2b519cf9126544107b3f27b98de9a/x/bank/module.go#L113
```
// ConsensusVersion defines the current x/bank module consensus version.
const ConsensusVersion = 4
...
func (am AppModule) RegisterServices(cfg module.Configurator) {
	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)

	m := keeper.NewMigrator(am.keeper.(keeper.BaseKeeper), am.legacySubspace)
	if err := cfg.RegisterMigration(types.ModuleName, 1, m.Migrate1to2); err != nil {
		panic(fmt.Sprintf("failed to migrate x/bank from version 1 to 2: %v", err))
	}

	if err := cfg.RegisterMigration(types.ModuleName, 2, m.Migrate2to3); err != nil {
		panic(fmt.Sprintf("failed to migrate x/bank from version 2 to 3: %v", err))
	}

	if err := cfg.RegisterMigration(types.ModuleName, 3, m.Migrate3to4); err != nil {
		panic(fmt.Sprintf("failed to migrate x/bank from version 3 to 4: %v", err))
	}
}
...
func (AppModule) ConsensusVersion() uint64 { return ConsensusVersion }
```

Reply:
Documentation of the check part of the update
https://github.com/cosmos/cosmos-sdk/blob/main/docs/core/upgrade.md

Reply:
@hikaruNagamine 
Does downgrading it from 2 to 1 cause any problems?

Reply:
@mkXultra 
I have confirmed that we can upgrade and operate BankSend and other operations.
But I haven't checked the functionality in ununifi/x, so I don't know exactly if it is affected.
I'm sure I can upgrade without any changes, so I don't think it's an issue.

Reply:
ok